docs: update general graphcast docs, add a page for 3la radio (#63)
graphops/docs: docs/graphcast/radios/poi-radio.md

| `PERSISTENCE_FILE_PATH`    | Relative path. If set, the Radio will periodically store states of the program to the file in json format (off by default).                                                                                                        |
| `DISCV5_ENRS`              | Comma separated ENRs for Waku Discv5 bootstrapping. Defaults to empty list.                                                                                                                                                        |
| `DISCV5_PORT`              | Discoverable UDP port. Default: `9000`                                                                                                                                                                                             |

> | `ID_VALIDATION`            | Defines the level of validation for message signers used during radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer`. Default: `registered-indexer` |

Reviewer: ```suggestion
| `ID_VALIDATION`            | Defines the level of validation for message signers used during Radio operation. Options include: `no-check`, `valid-address`, `graphcast-registered`, `graph-network-account`, `registered-indexer`, `indexer`. Default: `registered-indexer` |
```

> ### Configurations explained

Reviewer: ```suggestion
### Configuration explained
```

- **on-chain**: Subscribe to on-chain topics and user-defined static topics. This is the default coverage level and is suitable for indexers who only want to compare data for deployments with active allocations.
- **minimal**: Only subscribe to user-defined static topics. This level is for Indexers who want to limit their participation to specific topics of interest.

#### Identity validaiton

> `ID_VALIDATION` is used to define level of validation for message signers used during radio operation. We recommend `registered-indexer` for most strict identity validation, while `indexer` and is a viable option for those who want to use the network before considering Grapchast ID registration. Choose sender identity validation for your radio base on your use case and security preference.

Reviewer: ```suggestion
`ID_VALIDATION` is used to define level of validation for message signers used during Radio operation. We recommend `registered-indexer` for most strict identity validation, while `indexer` is a viable option for those who want to use the network before considering Grapchast ID registration. You can choose a sender identity validation mechanism for your Radio, based on your use case and security preferences.
```

Available Options:
- **no-check**: No check on the message signature and do not verify the signer of messages. All messages should pass the sender check.
- **valid-address**: signer needs to be a valid Eth address. Messages should be traceable to an Ethers wallet.
- **graphcast-registered**: signer must be registered at Graphcast Registry.
- **graph-network-account**: signer must be a Graph account.
- **registered-indexer**: signer must be registered at Graphcast Registry and correspond to an Indexer satisfying the indexer minimum stake requirement.

> - **indexer**: signer must be registered at Graphcast Registry or is a Graph Account, and correspond to an Indexer satisfying the indexer minimum stake requirement.

Reviewer: same suggestions as above
